src/core/lib/iomgr/tcp_custom.h

```diff
@@ -24,6 +24,8 @@
 #include "src/core/lib/iomgr/endpoint.h"
 #include "src/core/lib/iomgr/sockaddr.h"
 
+#define GRPC_CUSTOM_SOCKET_OPT_SO_REUSEPORT (0x00000010u)
```

Comment:
How did you come up with this magic number? Since this value is being passed to `grpc_custom_socket_vtable->bind()`, I assume this value must be defined elsewhere, either in Core, or in the Linux kernel. In either case, there should be some way to use *that* symbol instead of adding a duplicate.

Reply:
Changed to use the kernel defined value: `0x0200`

https://github.com/torvalds/linux/blob/a2d79c7174aeb43b13020dd53d85a7aefdd9f3e5/arch/parisc/include/uapi/asm/socket.h#L18